test/SILGen/objc_properties.swift

```diff
@@ -290,10 +290,10 @@ class SomeWrapperTests {
 // CHECK-LABEL: sil hidden [ossa] @$s15objc_properties16SomeWrapperTestsCyACSScfc : $@convention(method) (@owned String, @owned SomeWrapperTests) -> @owned SomeWrapperTests {
 // CHECK:  [[M:%.*]] = function_ref @$s15objc_properties16SomeWrapperTestsC04someD0SivsTD
 // CHECK:  [[C:%.*]] = partial_apply [callee_guaranteed] [[M]]({{.*}})
-// CHECK: assign_by_wrapper {{%.*}}: $Int to {{%.*}} : $*SomeWrapper, init {{.*}} : $@callee_guaranteed (Int) -> SomeWrapper, set [[C]] : $@callee_guaranteed (Int) -> ()
+// CHECK: assign_by_wrapper origin property_wrapper, {{%.*}}: $Int to {{%.*}} : $*SomeWrapper, init {{.*}} : $@callee_guaranteed (Int) -> SomeWrapper, set [[C]] : $@callee_guaranteed (Int) -> ()
```

Comment:
kind of a trivial nit since it's made-up syntax, but there's an established convention for pretty-printing SIL instruction modifiers within `[]` brackets. It helps distinguish the instruction name from the modifier. Maybe something like `assign_by_wrapper [property_wrapper], ...` and `assign_by_wrapper [type_wrapper], ...` would be a better match. I know I gave praise for printing out the modifier above, but after seeing it without the brackets I had to mention it.

Reply:
I didn’t use [] because that means “optional” and original is not.